agent/cmd/cuda-checkpoint-helper/transfer_config.cpp

```
@@ -0,0 +1,284 @@
+      char suffix[32];
+      std::snprintf(suffix, sizeof(suffix), ".part-%04zu", index);
+      path += suffix;
+    }
+    storage->files.push_back({std::move(path), size});
+    storage->ranges.push_back({logical_offset, size, index, 0});
+    logical_offset += size;
+  }
+  return true;
+}
+
+std::string JsonEscape(std::string_view value) {
```

Cppcheck (2.21.0)

[style] The function 'JsonEscape' is never used.

(unusedFunction)